cli.js

```diff
@@ -155,6 +153,8 @@ function errorHandler(error) {
             console.error('Error: ' + itemError.error_code);
             console.error(itemError.error_detail);
         }
+    } else {
+        console.error('Error: ' + (error?.message || 'Unknown error'));
```

Comment:
What is this for? The first line of the function already outputs the full error, the extra conditions are to *add* information. Should be reverted